src/engagement_db_to_analysis/configuration.py

```diff
@@ -1,13 +1,22 @@
 from dataclasses import dataclass
 
+from core_data_modules.data_models import CodeScheme
+
 
 @dataclass
 class DatasetTypes:
     DEMOGRAPHIC = "demographic"
     RESEARCH_QUESTION_ANSWER = "research_question_answer"
 
+@dataclass
+class CodingConfiguration:
```

Comment:
Does calling this LabelConfiguration work here? Please leave as CodingConfiguration if you're not sure/disagree

Reply:
Not sure at this stage - let us leave it as CodingConfiguration and check back after the analysis pipeline matures - we might add more configs that mean otherwise.